packages/nx/src/command-line/add/add.ts

```diff
@@ -104,7 +107,18 @@ async function initializePlugin(pkgName: string): Promise<void> {
   spinner.start();
 
   try {
-    await runNxAsync(`g ${pkgName}:${initGenerator}`);
+    let updatePackageScripts: boolean;
+    if (options.updatePackageScripts !== undefined) {
+      updatePackageScripts = options.updatePackageScripts;
+    } else {
+      updatePackageScripts =
+        process.env.NX_PCV3 === 'true' && coreNxPlugins.includes(pkgName);
+    }
```

Comment:
@jaysoo @FrozenPandaz should we ignore `--updatePackageScripts` if no `NX_PCV3=true`?

Reply:
Let's keep it as is, we will be removing the flag later anyway.